Makefile

```diff
@@ -12,7 +12,7 @@ prepare-check:
 check:
 	@#`python3 -m pytest` doesn't work here b/c the way requre overrides import system:
 	@#`AttributeError: module 'importlib_metadata' has no attribute 'distributions'
-	PYTHONPATH=$(CURDIR) PYTHONDONTWRITEBYTECODE=1 pytest-3 --verbose --showlocals $(TEST_TARGET)
+	PYTHONPATH=$(CURDIR) PYTHONDONTWRITEBYTECODE=1 pytest --verbose --showlocals $(TEST_TARGET)
```

Comment:
Please, do not include this change here. Create a new pull-request / issue for that so others can say something to that topic. (We want to solve one thing in one PR.)

Reply:
okay